streams/src/main/java/org/apache/kafka/streams/kstream/Aggregator.java

```diff
@@ -18,13 +18,22 @@
 package org.apache.kafka.streams.kstream;
 
 /**
- * The Aggregator interface for aggregating values of the given key.
+ * The {@link Aggregator} interface for aggregating values of the given key.
  *
  * @param <K>   key type
  * @param <V>   original value type
  * @param <T>   aggregate value type
  */
 public interface Aggregator<K, V, T> {
 
+    /**
+     * Compute a new aggregate for the message with the given key and value and the current aggregate (ie,
+     * {@link #apply(Object, Object, Object)} "increments" an aggregate with another message).
```

Comment:
I don't understand:

> (ie, {@link #apply(Object, Object, Object)} "increments" an aggregate with another message)

Where does this example imply that the function is incrementing?

Also, typo: "ie" -> "i.e."